static/header.less

```diff
@@ -178,16 +178,14 @@
     @media screen and (max-width: 399px){
         display: none;
     }
-    padding: 8px 5px 10px;
     margin: 0 0 0 @gutter;
     list-style: none;
     justify-content: center;
     align-items: center;
+    padding: 0 5px;
+    position: relative;
     .dropdown {
         display: flex;
-        position: relative;
-        width: 85px;
-        height: 35px;
```

Comment:
85x35 isn’t the correct aspect ratio; I think it should’ve been 85x30, or made wider than 85.